app/views/pages/hiring_content.html.haml

```diff
@@ -0,0 +1,145 @@
+        Through our work we aim to improve the global standard of programming and increase participation in tech from under-represented groups.
+        You can learn more about Exercism in our #{link_to '"About Exercism" pages', 'https://exercism.org/about', target: '_blank', rel: :noopener}.
+      %p.text-p-xlarge.mb-16
+        At the heart of Exercism is our content - thousands of programming exercises across over 5o programming languages, designed to teach and enable practice of programming concepts, nearly all of which has been written by our volunteer community.
+        As we continue to grow our content, and plan to expand it beyond our traditional exercise format, we need someone to lead our content strategy.
+        This is a hands-on role, where you'll be writing and editor content, alongside planning and leading our volunteer maintainers.
+        Currently this role is split between the CEO and Head of Open Source, both of whom you'll work closely with.
+      %p.text-p-xlarge.mb-32
+        We are building an exceptional team of ambitious, passionate people who love their work, excel at what they do, and have fun doing it.
+        The values of empathy, integrity and personal-growth are core to everything we do.
+        If that sounds exciting to be part of, and you feel you have the skills to help, we'd love for you to apply.
+
+      %h2.text-h2.mb-12 Responsibilities
+      %p.text-p-xlarge.mb-16
+        Your role is to fundmentally take all the work currently done by the CEO and the Head of Open Source,
+        Your role splits roughly into three areas.
+      %p.text-p-xlarge.mb-12
+        The first is strategic, which has the following key responsibilities:
+      %ul.text-p-xlarge.mb-16
+        %li Work with our CEO, Head of Open Source to more tighly define our our wider content strategy for the next three years.
+        %li Work with our senior maintainers to design and document systems that improve the flow and quality of content.
+        %li Work with our content partners (e.g. Packt Publishing) to determine synergies between their content and ours, and how we can make the most use of their educational portfolio.
+      %p.text-p-xlarge.mb-12
+        The second is a leadership role:
+      %ul.text-p-xlarge.mb-16
+        %li You will be the senior content voice for our community, answering questions, arbitrating discussions, and acting as the knowledgable sage of wisdom. This doesn't mean you have to be the most knowledgable on any given subject, but means you must speak to Exercism's perspective on content, education, and how we make pragmatic decisions.
+        %li You will act as a locum maintainer - supporting tracks where there are no maintainers, addressing issues and reviewing pull-requests from contributors. In this you will be supported by our community "reviewers" team, who help out in PR reviews on tracks where there are no formal maintainers.
```

Comment:
If I liked all the other responsibilities, this one would cause me to not apply, because there's no way I would want to be responsible for PRs in potentially 50+ languages I don't know.

Reply:
So we need to decide if this is their job, or someone else's job, because currently it's work that Erik does, so either he needs to carry on doing it, or we need someone else to be doing it. @ErikSchierboom Your thoughts?

Reply:
Ehm, good question. 

First let me reply to:

> there's no way I would want to be responsible for PRs in potentially 50+ languages I don't know.

Currently, that's sort of what I'm doing, but in practice it is definitely not as scary as that as:

a) many tracks are updated only rarely
b) the well-maintained tracks usually require less intense reviews

That said, I can totally see that this might cause people to not want to be interested in the Content Factotum role, as it _does_ sound quite daunting.
I think if we add the caveat to the above-mentioned responsibilities that the Content Factotum is "only" responsible for support/issues/PRs if they related to content, it could become substantially less daunting.
Anything non-content related (= code) would then be my responsibility (at least for now).
This would mean that PRs for things like track tooling (representer/test runner/analyzer) would be my responsibility, but things like rewriting/reviewing an exercise's introduction would be the Content Factotum's.

We (Content Factotum and I as Head Open Source) probably need to work together lots if we go this route, but I think that's fine.

Reply:
Yeah, that's probably a nicer way of word it, to say "Together, you and the Head of Open Source will act as a locum maintainer..."